vividus-plugin-web-app/src/main/java/org/vividus/bdd/steps/ui/web/ElementSteps.java

```diff
@@ -193,10 +193,10 @@ public void doesEachElementByLocatorHaveChildWithLocator(Locator elementLocator,
     @Then("the context element has the CSS property '$cssName'='$cssValue'")
     public void doesElementHaveRightCss(String cssName, String cssValue)
     {
-        WebElement element = uiContext.getSearchContext(WebElement.class);
-        String actualCssValue = webElementActions.getCssValue(element, cssName);
-        descriptiveSoftAssert.assertEquals("Element has correct css property value", cssValue,
-                actualCssValue);
+        uiContext.getSearchContext(WebElement.class)
+                .map(element -> webElementActions.getCssValue(element, cssName))
+                .ifPresent(actualCssValue -> descriptiveSoftAssert.assertEquals(
```

Comment:
Null could be returned by getCssValue, but won't be validated

Reply:
need to check whether `getCssValue` returns `null` in case of missing css property

Reply:
Just checked it, returned empty string, so we could consider it as resolved